Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -631,9 +631,11 @@ def install_app(self, app_path, env=None):
             if exit_code == 0:
                 return True
 
+            # Return code 18 indicates that the app is not compatible with the current device, which can
+            # happen under load and may not occur on retry.
```

Comment:
Mentioning a radar to track this behavior would be helpful (unless we're okay committing to it forever).